tensorflow/c/experimental/gradients/nn_grad_testutil.cc

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Could we define `BiasAddGradModel` in `nn_grad_test.cc` and not have `nn_grad_testutil.*`?

Reply:
Done